bciers/apps/reporting/src/app/components/routes/operations/Page.tsx

```diff
@@ -1,17 +1,24 @@
 import { getReportingYear } from "@reporting/src/app/utils/getReportingYear";
 import Operations from "../../operations/Operations";
 import { OperationsSearchParams } from "../../operations/types";
+import { getReportingDueDate } from "@reporting/src/app/utils/getReportingDueDate";
 
 export default async function OperationsPage({
   searchParams,
 }: {
   searchParams: OperationsSearchParams;
 }) {
+  const reportingDueDate = await getReportingDueDate();
   const reportingYear = await getReportingYear();
```

Comment:
Once we've grouped these into a single API call and endpoint, it'll be much more efficient to fetch the data in one go here